### DISCUSSION TITLE
Can I do this?

### DISCUSSION THREAD

**Discussion by xiatian12345** [Microsoft/node-pty#629] [Q&A]
## Environment details

- OS:
- OS version:
- node-pty version:

## Issue description
Can I do this to get the result of a command after it has finished executing, as well as get a notification after the command has been entered?

**Comment by Tyriar** [comment]
There are a bunch of discussions in the issues about this, tldr: not really, you need shell integration or to launch separate shells per thing you want the exit status reported on.
